lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp

```diff
@@ -2687,7 +2687,7 @@ uint64_t SymbolFileDWARF::GetDebugInfoSize() {
     if (cu == nullptr)
       continue;
 
-    SymbolFileDWARFDwo *dwo = cu->GetDwoSymbolFile();
+    SymbolFileDWARFDwo *dwo = cu->GetDwoSymbolFile(false);
```

Comment:
Instead of hard-coding `false` here, I think it should the caller of `SymbolFileDWARF::GetDebugInfoSize` caller (statistics.cpp in this case) making this decision.

Reply:
Will put a new PR to address this